docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -11,14 +11,13 @@ There you can find links to upgrade notes for other versions too.
 if you want to have products data exported to Elasticsearch after `build-demo` target is run ([#824](https://github.com/shopsys/shopsys/pull/824/files))
 
 ### Application
-- moved data fixtures to project-base [#854](https://github.com/shopsys/shopsys/pull/854)
-    - moving data fixtures was done in this [#854](https://github.com/shopsys/shopsys/pull/854) so you can get inspiration how to do it
-        - you need to move all data fixtures to a new folder `DataFixtures` in `ShopBundle`
+- copy data fixtures into your project namespace `ShopBundle\DataFixtures` from `FrameworkBundle\DataFixtures` of [shopsys/framework] repository ([#854](https://github.com/shopsys/shopsys/pull/854))
+    - moving data fixtures was done in [#854](https://github.com/shopsys/shopsys/pull/854) so you can get inspiration how to do it
         - move configuration file `Resources/config/services/data_fixtures.yml` and import it in `services.yml` like this:
```

Comment:
move it where ?

Reply:
The information was added in later commits